src/platform_impl/orbital/mod.rs

```diff
@@ -151,38 +151,13 @@ impl MonitorHandle {
         1.0 // TODO
     }
 
-    pub fn current_video_mode(&self) -> Option<VideoModeHandle> {
+    pub fn current_video_mode(&self) -> Option<VideoMode> {
         // (it is guaranteed to support 32 bit color though)
-        Some(VideoModeHandle { monitor: self.clone() })
-    }
-
-    pub fn video_modes(&self) -> impl Iterator<Item = VideoModeHandle> {
-        self.current_video_mode().into_iter()
-    }
-}
-
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
-pub struct VideoModeHandle {
-    monitor: MonitorHandle,
-}
-
-impl VideoModeHandle {
-    pub fn size(&self) -> PhysicalSize<u32> {
-        // TODO
-        PhysicalSize::default()
-    }
-
-    pub fn bit_depth(&self) -> Option<NonZeroU16> {
-        None
-    }
-
-    pub fn refresh_rate_millihertz(&self) -> Option<NonZeroU32> {
-        // TODO
         None
     }
 
-    pub fn monitor(&self) -> MonitorHandle {
-        self.monitor.clone()
+    pub fn video_modes(&self) -> impl Iterator<Item = VideoMode> {
+        std::iter::empty()
```

Comment:
This returned the `iter::Empty` in Android :sweat_smile:. Planned for changing the type later?

Reply:
I don't understand. The plan is to remove `impl Iterator` in general from everything though.

Reply:
Asking why you don't:
```suggestion
    pub fn video_modes(&self) -> std::iter::Empty<VideoMode> {
        std::iter::empty()
```

Unless you want to return another (non-empty...) iterator somewhere down the line without breaking the API?

Reply:
I guess API "doesn't matter" because it's all wrapped via `self.inner.video_modes()` in the end?

Reply:
It's all internal, yes.